incompatibility/plugin-creation.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Still needed as you move forward with one plugin?

Reply:
moved so that plugin-creation works with gnostic-grpc, will be using for updating executable during development.